stable/influxdb/templates/deployment.yaml

```diff
@@ -50,13 +50,13 @@ spec:
         {{- end }}
         livenessProbe:
           httpGet:
-            path: /ping
+            path: {{ .Values.deployment.probePath }}
             port: api
           initialDelaySeconds: 30
           timeoutSeconds: 5
```

Comment:
Should we remove hard code and make replace this with  {{ .Values.livenessProbe.timeoutSeconds | default 5}}